nomic/atlas.py

```diff
@@ -526,7 +530,7 @@ def create_index(self, project_id: str, index_name: str, indexed_field=None, col
         to_return['project_id'] = project['id']
         return CreateIndexResponse(**to_return)
 
-    def add_text(self, project_id: str, data: List[Dict], shard_size=1000, num_workers=10, pbar=None):
+    def add_text(self, project_id: str, data: List[Dict], shard_size=1000, num_workers=10, replace_empty_values_with_null=True, pbar=None):
```

Comment:
can we make this replace_empty_strings_with_string_null? as its written its confusing.